README.md

```diff
@@ -1 +1,84 @@
+
+![Addresses](./diagrams/addresses.svg)
+
+###### Transactions
+
+Transactions table contains your transaction history. You can keep refreshing to monitor the status of your transactions. Your transaction status could be as follows.
+
+1. `New`: The transaction has been received and is waiting for the fund transaction to be mined.
+2. `Sent`: The transaction has been broadcast to the network and is waiting for it to be mined.
+3. `Confirmed`: The transaction has been mined in a block and transaction execution is successful on-chain.
+4. `Failed`: The transaction has been mined in a block and transaction execution is failed on-chain. This is likely because you sent your transaction to a smart contract, and part of the contract logic decided to reject any state transition.
+
+![Transactions](./diagrams/transactions.svg)
+
+###### Account History
+
+Account History table displays a history of your account balance change.
+
+![Account History](./diagrams/account-history.svg)
+
+#### Using FuelStop's RPC URL
+
+FuelStop supports all [standard Ethereum JSON-RPC APIs](https://eth.wiki/json-rpc/API), which means it is able to work seemlessly with popular libraries such as [web3.js](https://github.com/ethereum/web3.js/) and [ethers.js](https://github.com/ethers-io/ethers.js/), and tools such as [Metamask](https://metamask.io/) and [Remix Ethereum IDE](https://remix.ethereum.org/).
+
+###### web3.js
+
+```javascript
+const Web3 = require('web3');
+const web3 = new Web3(url); // url is your FuelStop project RPC URL
+```
+
+###### ethers.js
+
+```javascript
+const ethers = require('ethers');
+const provider = new ethers.providers.JsonRpcProvider(url); // url is your FuelStop project RPC URL
+```
+
+###### Metamask
+
+Go to `Settings` -> `Networks` -> `Add Network`, and then fill in the "Network Name" of your choice and your FuelStop RPC URL, and your Metamask is connected.
+
+![Metamask](./diagrams/metamask.svg)
+
+If you have loaded the private key in Metamask of some addresses that you also registered in FuelStop, you may notice that it's ETH balacne shown in Metamask does not equal to its actual balance on the Ethereum network. This is because FuelStop intercepts the RPC calls that Metamask uses to query your balance when it sees those calls are for an address registered for the corresponding project. The balance after interception is your `actual ETH balance` + `equivalent ETH amount of FuelStop project balance`. This effectively would allow Metamask to work with FuelStop seemlessly when it comes to sending transactions, and is useful for you to continue using Metamask to monitor your "ETH" balance.
```

Comment:
```suggestion
If you register a address to Fuelstop that is associated with one of your Metmask accounts, you may notice that that the ETH balance shown in Metamask is not equal to its actual balance on the Ethereum network. This is because FuelStop intercepts the RPC calls that Metamask uses to query your balance. When FuelStop sees a call querying for a balance of a registered address it will give back your `actual ETH balance` + `the equivalent ETH amount of your FuelStop project balance`. This allows Metamask to work with FuelStop seamlessly when it comes to sending transactions, and is useful for you to continue using Metamask to monitor your "ETH" balance.
```